Pod/Classes/URBNExtraTextHighlightedTextStorage.h

```diff
@@ -0,0 +1,15 @@
+//
+//  URBNExtraTextHighlightedTextStorage.h
+//  Pods
+//
+//  Created by Nick DiStefano on 6/19/15.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface URBNExtraTextHighlightedTextStorage : NSTextStorage
+
+- (instancetype)initWithErrorTextColor:(UIColor *)errorTextColor maxLength:(NSInteger)maxLength;
```

Comment:
Along with expanding the above category for on the fly updates to ranges and colors, you may want to do the same down here. And expose either properties or methods.